static/app/components/searchQueryBuilder/index.tsx

```diff
@@ -297,7 +297,7 @@ export function SearchQueryBuilder({
         onBlur={() =>
           onBlur?.(state.query, {parsedQuery, queryIsValid: queryIsValid(parsedQuery)})
         }
-        ref={wrapperRef}
+        ref={wrapperRef as React.RefObject<HTMLInputElement>}
```

Comment:
`Input` explicitly states `ref` as `Ref<HTMLInputElement>`, and `.withComponent` doesn’t seem to be able to overwrite that. I hope this gets better with React 19 without `forwardRef`, but for now, this type assertion is necessary because at runtime, this will be a `Ref` of a `div`.